controller/array_action/errors.py

```diff
@@ -99,6 +99,12 @@ def __init__(self, pool, array):
         self.message = messages.PoolDoesNotExist_message.format(pool, array)
 
 
+class PoolParameterIsMissing(BaseArrayActionException):
+
+    def __init__(self, array_type):
+        self.message = messages.PoolIsRequired.format(array_type)
```

Comment:
message const name should be aligned to exception class name

Reply:
done